### DISCUSSION TITLE
2022-10-29-my-first-post/

### DISCUSSION THREAD

**Discussion by giscus** [yzxiu/yzxiu.github.io#3] [Announcements]
# 2022-10-29-my-first-post/

一、配置 参考：https://hugoloveit.com/zh-cn/posts/ 二、图片 2.1 图片上传 typora配置： typora setting 图片插入： 1 2 #

https://yzxiu.github.io/2022-10-29-my-first-post/

**Comment by yzxiu** [comment]
# good!
